bin/entrypoint.sh

```diff
@@ -66,11 +81,25 @@ ARKMANAGER="$(command -v arkmanager)"
 cd "${ARK_SERVER_VOLUME}"
 
 echo "Setting up folder and file structure..."
-create_missing_dir "${ARK_SERVER_VOLUME}/log" "${ARK_SERVER_VOLUME}/backup" "${ARK_SERVER_VOLUME}/staging" "${ARK_SERVER_VOLUME}/template"
-copy_missing_file "${STEAM_HOME}/arkmanager.cfg" "${ARK_SERVER_VOLUME}/template/arkmanager.cfg"
-copy_missing_file "${STEAM_HOME}/crontab" "${ARK_SERVER_VOLUME}/template/crontab"
-# copy_missing_file "${ARK_SERVER_VOLUME}/template/arkmanager.cfg" "${ARK_SERVER_VOLUME}/arkmanager.cfg"
-copy_missing_file "${ARK_SERVER_VOLUME}/template/crontab" "${ARK_SERVER_VOLUME}/crontab"
+create_missing_dir "${ARK_SERVER_VOLUME}/log" "${ARK_SERVER_VOLUME}/backup" "${ARK_SERVER_VOLUME}/staging"
+
+
+echo "Setting up Arkmanager..."
+# setup arkmanager directories
+if [[ ! -d ${ARK_TOOLS_DIR} ]]; then
+  sudo mv "/etc/arkmanager" "${ARK_TOOLS_DIR}"
+  ensure_rights "${ARK_TOOLS_DIR}"
+  rm -f "${ARK_TOOLS_DIR}/arkmanager.cfg" "${ARK_TOOLS_DIR}/instances/main.cfg"
+fi
+
+# symlink arkmanager directories
+sudo rm -rf "/etc/arkmanager"
+sudo ln -s "${ARK_TOOLS_DIR}" "/etc/arkmanager"
```

Comment:
I love this. This opens up the out-of-the-box functionality for uses to define their own config. This also fixes a problem that I discovered that has been unmentioned. Specifying a different ports for the server to use was broken before because the arkmanager.cfg that they were listed in was not loading on server startup. This should fix it, but I will test here in a min.